### ISSUE TITLE
[java] DoNotUseThreads report duplicate warnings

### ISSUE THREAD

**Issue by Belle-PL** [pmd/pmd#4210]
**Affects PMD Version:6.51.0**

**Rule: DoNotUseThreads**

Please provide the rule name and a link to the rule documentation:
<https://pmd.github.io/latest/pmd_rules_java_multithreading.html#donotusethreads>

**Description:**
In the following code example, line 2 uses the class `Thread` which should be forbiden and it reports two duplicate warnings.

**Code Sample demonstrating the issue:**

```java
public final class Test {  
    public final Thread BAR = new Thread();  // report two duplicate warnings in this line
   
    public Thread getBAR() {  // report one warning in this line
        return this.BAR;
    }
}
```

**Expected outcome:**

PMD reports two duplicate warnings at line 2, but one warning is appropriate.

**Running PMD through:** *CLI*

**Comment by Belle-PL** [comment]
Sorry, the label should be false positive or duplicate warning, not false negative. 🥲
